### ISSUE TITLE
Migration to version 10.0

### ISSUE THREAD

**Issue by pedrobaeza** [OCA/vertical-education#3]
# Todo

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0
# Modules to migrate
- [ ] openeducat_achievement
- [ ] openeducat_activity
- [ ] openeducat_admission
- [ ] openeducat_alumni
- [ ] openeducat_assignment
- [ ] openeducat_attendance
- [ ] openeducat_classroom
- [ ] openeducat_core > education - By @angelmoya - #4 
- [ ] openeducat_erp
- [ ] openeducat_exam
- [ ] openeducat_facility
- [ ] openeducat_fees
- [ ] openeducat_health
- [ ] openeducat_hostel
- [ ] openeducat_l10n_in
- [ ] openeducat_l10n_in_admission
- [ ] openeducat_library
- [ ] openeducat_parent
- [ ] openeducat_placement
- [ ] openeducat_scholarship
- [ ] openeducat_timetable
- [ ] openeducat_transportation

**Comment by lasley** [comment]
I'll claim core v10 if someone will mark please. Is there anything special I should know before diving into this repo?

**Comment by pedrobaeza** [comment]
There's a "fork" (in fact, this one is a fork of it instead) in https://github.com/openeducat. You can start from it and maybe join efforts, but bring it this to OCA.

**Comment by lasley** [comment]
Interesting, thanks @pedrobaeza. Any reason why there is a divergent fork? Did the original just keep getting developed after we moved into OCA or something?

**Comment by pedrobaeza** [comment]
Yeah, they continue in their branch and never come back again to OCA. I don't know the reasons.

**Comment by lasley** [comment]
That's sad. Feels like vertical-medical all over again 😦

**Comment by lasley** [comment]
I'll ponder hard, I may help out here. We have some interesting things happening with school districts in the US right now, may be an opportunity for my SaaS

**Comment by pedrobaeza** [comment]
OK, you know I support all your efforts to bring to OCA the maximum :wink:

Talk to them anyway to see what are their reasons.

**Comment by angelmoya** [comment]
Hi @pedrobaeza and @lasley,

I check openeducat repository, and created an Issue because is not implemented multicompany on any object. Maintainer of this project said that this functionality is supported on enterprise version. So this is why they don't update on OCA.

It is quite possible that early I will start to work on this project.

Other questión, as it is a fork, What do you think about change name of modules from openeducat_xxx to education_xxx?

**Comment by lasley** [comment]
> Other questión, as it is a fork, What do you think about change name of modules from openeducat_xxx to education_xxx?

Definitely. This is exactly what we did in `vertical-medical`, which used to be `oemedical_` and is now `medical_`. This solution seconds as a really great way to tell which modules have been redesigned.

**Comment by pedrobaeza** [comment]
OK for me too. This repository can have a lot of potential with someone of you doing right the things :wink:

**Comment by lasley** [comment]
I was thinking about starting some dev here too, so it's nice to know I wouldn't be taking on the whole thing like I did in vertical medical. 

Education has a deep place in my heart, even if I personally hated school 🚀 

@angelmoya - How drastic of changes are you thinking? This repo's pretty out of date, so we could be quite drastic due to not really having to worry about an upgrade path.

**Comment by lasley** [comment]
Also it might be worth considering that we merge recent work in their fork in to this one before starting our work. I don't see any reason why that's not a possibility, as long as we maintain attribution (why wouldn't we)

**Comment by angelmoya** [comment]
Yes, I think that It's good idea to update this code.

There are a lot of work to do, I think that core is good, but a lot of relations and views are missing, I think because they implement on enterprise version. For example
- no multiconpany
- on students you can not view courses
- on batch you can not view students

A lot of things that could be improved.

**Comment by angelmoya** [comment]
Hi @lasley and @pedrobaeza , finally I just start project related with this repository.
I start fork on this PR #4 , I don't know if there is the correct way to do it.

I start with openeducat_core (renamed as education) module.

- I update last commits from openeducat.
- Delete modules openeducat_erp, web_openeducat and openeducat_support
- Delete images and links to openeducat

License is LGPL, can we change it on this fork?

**Comment by lasley** [comment]
> License is LGPL, can we change it on this fork?

Yup any OSI license compatible with Odoo's is good for OCA (so basically any of the *GPL-3)
